<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDashboard</class>
 <widget class="QWidget" name="MainDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#MainDashboard {
    background-color: #303030;
}
QLabel {
    color: white;
}
QPushButton {
    background-color: #404040;
    color: white;
    border: 2px solid #505050;
    border-radius: 10px;
    padding: 10px;
}
QPushButton:hover {
    background-color: #505050;
    border-color: #606060;
}
QPushButton:pressed {
    background-color: #606060;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>40</number>
   </property>
   <property name="leftMargin">
    <number>80</number>
   </property>
   <property name="topMargin">
    <number>80</number>
   </property>
   <property name="rightMargin">
    <number>80</number>
   </property>
   <property name="bottomMargin">
    <number>80</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="logoLayout">
     <item>
      <spacer name="logoLeftSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="logoLabel">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>:/Images/SandDriveLogo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>450</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1200</width>
         <height>450</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="logoRightSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="welcomeLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>48</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Welcome</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="actionButtons">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="scansButton">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Scans</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="logsButton">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Logs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reportsButton">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Reports</string>
       </property>
      </widget>
     </item>
         <item>
            <widget class="QPushButton" name="createUserButton">
             <property name="minimumSize">
                <size>
                 <width>180</width>
                 <height>60</height>
                </size>
             </property>
             <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <bold>true</bold>
                </font>
             </property>
             <property name="text">
                <string>Create User</string>
             </property>
            </widget>
         </item>
         <item>
            <widget class="QPushButton" name="deleteUserButton">
             <property name="minimumSize">
                <size>
                 <width>180</width>
                 <height>60</height>
                </size>
             </property>
             <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <bold>true</bold>
                </font>
             </property>
             <property name="text">
                <string>Delete User</string>
             </property>
            </widget>
         </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="logoutButton">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
